src/styles.ts

```diff
@@ -11,6 +11,18 @@ const GlobalStyles = createGlobalStyle`
     font-weight: 300;
   }
 
+  a {
```

Comment:
why are we adding doing global styles here? is this in scope of the PR?

Reply:
links are unstyled right now, so they're purple / navy / whatever the browser sets them to. The inline link used for this feature is an <a tag.

I think it's reasonable to have a base style for a tags